src/applications/personalization/preferences/tests/containers/PreferencesWidget.unit.spec.jsx

```diff
@@ -26,6 +28,8 @@ describe('<PreferencesWidget>', () => {
   });
   it('should render view with preferences', () => {
     props.preferences.dashboard['education-training'] = true;
+    props.setDismissedBenefitAlerts = () => true;
+    props.preferences.dismissedBenefitAlerts = [];
```

Comment:
I will add more tests for the alerts and the button as part of the button regression fix PR.